plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list/payment-gateway-list.tsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * External dependencies
+ */
+import { PaymentProvider } from '@woocommerce/data';
+import { getAdminLink } from '@woocommerce/settings';
+import { Gridicon } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DefaultDragHandle,
+	SortableContainer,
+	SortableItem,
+} from '~/settings-payments/components/sortable';
+import { PaymentExtensionSuggestionListItem } from '~/settings-payments/components/payment-extension-suggestion-list-item';
+import { PaymentGatewayListItem } from '~/settings-payments/components/payment-gateway-list-item';
+import './payment-gateway-list.scss';
+
+interface PaymentGatewayListProps {
+	providers: PaymentProvider[];
+	installedPluginSlugs: string[];
+	installingPlugin: string | null;
+	setupPlugin: ( id: string, slug: string ) => void;
+	updateOrdering: ( providers: PaymentProvider[] ) => void;
+}
+
+export const PaymentGatewayList = ( {
+	providers,
+	installedPluginSlugs,
+	installingPlugin,
+	setupPlugin,
+	updateOrdering,
+}: PaymentGatewayListProps ) => {
+	return (
+		<SortableContainer< PaymentProvider >
+			items={ providers }
+			className={ 'settings-payment-gateways__list' }
+			setItems={ updateOrdering }
+		>
+			{ providers.map( ( provider: PaymentProvider ) => {
+				switch ( provider._type ) {
+					case 'suggestion':
+						const pluginInstalled = installedPluginSlugs.includes(
+							provider.plugin.slug
+						);
+						return (
+							<SortableItem
+								key={ provider.id }
+								id={ provider.id }
+							>
+								{ PaymentExtensionSuggestionListItem( {
+									extension: provider,
+									installingPlugin,
+									setupPlugin,
+									pluginInstalled,
+								} ) }
+							</SortableItem>
+						);
+					case 'gateway':
+						return (
+							<SortableItem
+								key={ provider.id }
+								id={ provider.id }
+							>
+								{ PaymentGatewayListItem( {
+									gateway: provider,
+								} ) }
+							</SortableItem>
+						);
+					case 'offline_pms_group':
+						return (
+							<SortableItem
+								key={ provider.id }
+								id={ provider.id }
+							>
+								<div
+									id={ provider.id }
+									className="transitions-disabled woocommerce-list__item woocommerce-list__item enter-done"
```

Comment:
`woocommerce-list__item` is typed twice here

I believe we should get `clickable-list-item` back here (see PR review comment)

and
onClick: () => {
  window.location.href = getAdminLink(
	  'admin.php?page=wc-settings&tab=checkout&section=offline'
  );
  },

as it was in `plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx` in this PR in line 101 (sorry, can't get direct link into that)

Reply:
Thank you for spotting this, great catch! I addressed it and it should work as described now.

Reply:
Big nit pick, but do you think we could vertically center the offline PMs group row arrow (like in the designs)?

Reply:
addressed in https://github.com/woocommerce/woocommerce/pull/53301/commits/7ec48d27d891852eeb75d6cb9017f125106d1dfb!

Reply:
<img width="1546" alt="image" src="https://github.com/user-attachments/assets/88fceda3-c145-488f-83da-4ac3e446de00">